frontend/src/metabase/query_builder/components/data-search/SearchResults.jsx

```diff
@@ -9,10 +9,22 @@ import Search from "metabase/entities/search";
 
 import { SearchResultItem } from "./SearchResultItem";
 
-export function SearchResults({ searchQuery, onSelect, databaseId }) {
+const propTypes = {
+  databaseId: PropTypes.string,
+  searchQuery: PropTypes.string.required,
+  onSelect: PropTypes.func.required,
+  searchModels: PropTypes.arrayOf(PropTypes.string.isRequired),
```

Comment:
nip: should we maybe list the `searchModels` options right away if we know what's in use now?
Like `PropTypes.arrayOf(PropTypes.oneOf(["card", "table"]))`